jsk_perception/sample/sample_imagenet_object_recognition.launch

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
you mean you don't want to remove `sample_alexnet_object_recognition.launch`?
What we want to do is adding `sample_imagenet_object_recognition.launch` with arg `network`